src/main/scala/no/ndla/crowdingateway/service/ConverterService.scala

```diff
@@ -10,17 +10,20 @@ package no.ndla.crowdingateway.service
 import no.ndla.crowdingateway.model.api.TranslationResponse
 import no.ndla.crowdingateway.model.domain.AddFileResponse
 
+import scala.util.{Success, Try}
+
 
 trait ConverterService {
   val converterService: ConverterService
 
   class ConverterService {
-    def asTranslationResponse(x: AddFileResponse): TranslationResponse = {
-      TranslationResponse(
+    def asTranslationResponse(x: AddFileResponse): Try[TranslationResponse] = {
```

Comment:
Forsåvidt et godt poeng. Jeg sa vel noe lignende om en converter du skrev for en stund tilbake, så der fikk jeg den 😄  Converterne burde være såpass enkle at det ikke skal være en behov for at de returnerer en `Try`